include/grpc/impl/codegen/atm_gcc_atomic.h

```diff
@@ -85,6 +85,11 @@ static __inline int gpr_atm_rel_cas(gpr_atm *p, gpr_atm o, gpr_atm n) {
       p, &o, n, 0, __ATOMIC_RELEASE, __ATOMIC_RELAXED));
 }
 
+static __inline int gpr_atm_full_cas(gpr_atm *p, gpr_atm o, gpr_atm n) {
```

Comment:
Yeah happy to take an AI to globally search replace atm_full with atm_acq_rel, but right now consistency wins for me (also I'd need to do some planning for such a rename as technically this is public API)